src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V62/Eth62ProtocolHandler.cs

```diff
@@ -258,41 +261,28 @@ protected void Handle(TransactionsMessage msg)
         {
             IList<Transaction> iList = msg.Transactions;
 
-            if (iList is ArrayPoolList<Transaction> apl)
-            {
-                HandleFast(apl.AsSpan());
-            }
-            else if (iList is Transaction[] array)
-            {
-                HandleFast(array.AsSpan());
-            }
-            else if (iList is List<Transaction> list)
-            {
-                HandleFast(CollectionsMarshal.AsSpan(list));
-            }
-            else
-            {
-                HandleSlow(iList);
-            }
+            BackgroundTaskScheduler.ScheduleTask((iList, 0), HandleSlow);
         }
 
-        private void HandleFast(Span<Transaction> transactions)
+        private Task HandleSlow((IList<Transaction>, int) request, CancellationToken cancellationToken)
```

Comment:
Name the ValueTuple params so can use `.startIndex` rather than `.Item2`
```suggestion
        private Task HandleSlow((IList<Transaction> txs, int startIndex) request, CancellationToken cancellationToken)
```